homeassistant/components/media_player/dlna_dmr.py

```diff
@@ -190,7 +190,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     if config.get(CONF_URL) is not None:
         url = config.get(CONF_URL)
-        name = config.get(CONF_NAME)
+        name = config.get(CONF_NAME, DEFAULT_NAME)
```

Comment:
No need to pass DEFAULT_NAME, voluptuous will have already set it to DEFAULT_NAME